src/PolyType/Abstractions/IUnionCaseShape.cs

```diff
@@ -23,6 +23,11 @@ public interface IUnionCaseShape
     /// </remarks>
     int Tag { get; }
 
+    /// <summary>
+    /// Gets a value indicating whether <see cref="Tag"/> is inferred or explicitly specified in a more stable way.
```

Comment:
Could you update the doc per https://github.com/eiriktsarpalis/PolyType/pull/126/files#r1979964214?